'librdkafka/rdkafka.h' file not found

```shell
$ sbcl
This is SBCL 2.0.0, an implementation of ANSI Common Lisp.
More information about SBCL is available at <http://www.sbcl.org/>.

SBCL is free software, provided as is, with absolutely no warranty.
It is mostly in the public domain; some portions are provided under
BSD-style licenses.  See the CREDITS and COPYING files in the
distribution for more information.
* (ql:quickload 'cl-rdkafka)
To load "cl-rdkafka":
  Load 1 ASDF system:
    cl-rdkafka
; Loading "cl-rdkafka"

[package cl-rdkafka/low-level]; clang -o /Users/noname007/.cache/common-lisp/sbcl-2.0.0-macosx-x64/Users/noname007/quicklisp/local-projects/cl-rdkafka/src/low-level/librdkafka-grovel__grovel-tmpGW0QV22K.o -c -g -Wall -Wundef -Wsign-compare -Wpointer-arith -O3 -g -Wall -O2 -fdollars-in-identifiers -DOS_THREAD_STACK -mmacosx-version-min=10.15 -D_DARWIN_USE_64_BIT_INODE -arch x86_64 -fno-omit-frame-pointer -fPIC -I/Users/noname007/quicklisp/dists/quicklisp/software/cffi_0.20.1/ /Users/noname007/.cache/common-lisp/sbcl-2.0.0-macosx-x64/Users/noname007/quicklisp/local-projects/cl-rdkafka/src/low-level/librdkafka-grovel__grovel.c
/Users/noname007/.cache/common-lisp/sbcl-2.0.0-macosx-x64/Users/noname007/quicklisp/local-projects/cl-rdkafka/src/low-level/librdkafka-grovel__grovel.c:6:10: fatal error:
      'librdkafka/rdkafka.h' file not found
#include <librdkafka/rdkafka.h>
         ^~~~~~~~~~~~~~~~~~~~~~
1 error generated.

debugger invoked on a CFFI-GROVEL:GROVEL-ERROR in thread #<THREAD "main thread" RUNNING {10005184C3}>: Subprocess #<UIOP/LAUNCH-PROGRAM::PROCESS-INFO {10037CE5A3}>
 with command ("clang" "-o" "/Users/noname007/.cache/common-lisp/sbcl-2.0.0-macosx-x64/Users/noname007/quicklisp/local-projects/cl-rdkafka/src/low-level/librdkafka-grovel__grovel-tmpGW0QV22K.o" "-c" "-g" "-Wall" "-Wundef" "-Wsign-compare" "-Wpointer-arith" "-O3" "-g" "-Wall" ...)
 exited with error code 1

Type HELP for debugger help, or (SB-EXT:EXIT) to exit from SBCL.

```

```shell
$ clang -o /Users/noname007/.cache/common-lisp/sbcl-2.0.0-macosx-x64/Users/noname007/quicklisp/local-projects/cl-rdkafka/src/low-level/librdkafka-grovel__grovel-tmpGHU3ALSV.o -c -g -Wall -Wundef -Wsign-compare -Wpointer-arith -O3 -g -Wall -O2 -fdollars-in-identifiers -DOS_THREAD_STACK -mmacosx-version-min=10.15 -D_DARWIN_USE_64_BIT_INODE -arch x86_64 -fno-omit-frame-pointer -fPIC -I/Users/noname007/quicklisp/dists/quicklisp/software/cffi_0.20.1/ /Users/noname007/.cache/common-lisp/sbcl-2.0.0-macosx-x64/Users/noname007/quicklisp/local-projects/cl-rdkafka/src/low-level/librdkafka-grovel__grovel.c
/Users/noname007/.cache/common-lisp/sbcl-2.0.0-macosx-x64/Users/noname007/quicklisp/local-projects/cl-rdkafka/src/low-level/librdkafka-grovel__grovel.c:11:7: warning: unused
      variable 'autotype_tmp' [-Wunused-variable]
  int autotype_tmp;
      ^
1 warning generated.
```

--- a/src/low-level/librdkafka-grovel.lisp
+++ b/src/low-level/librdkafka-grovel.lisp
@@ -17,6 +17,9 @@
 
 (in-package #:cl-rdkafka/low-level)
 
+#+darwin 
+(cc-flags "-I" "/usr/local/include")
+
 (include "librdkafka/rdkafka.h")
 
 (ctype size-t "size_t")

